UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Models/Request/ProductDataSinceDateTimeRequest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Swashbuckle.AspNetCore.Annotations;
+using System.Diagnostics.CodeAnalysis;
+
+namespace UKHO.ExchangeSetService.Common.Models.Request
+{
+    [ExcludeFromCodeCoverage]
```

Comment:
```suggestion
    [ExcludeFromCodeCoverage]
```

Although I do not expect unit tests for the POCOs we should not exclude them by default sine these are used as the input data for our controllers. they should get implicit coverage

which lets us identify if there are any edge cases we might have missed

Reply:
We have Removed ExcludeFromCodeCoverage.